trading_calendars/exchange_calendar_xwbo.py

```diff
@@ -0,0 +1,126 @@
+#
+# Copyright 2018 Quantopian, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from datetime import time
+
+from pandas.tseries.holiday import Holiday, GoodFriday, EasterMonday
+from pytz import timezone
+
+from .common_holidays import (
+    new_years_day,
+    epiphany,
+    ascension_day,
+    whit_monday,
+    corpus_christi,
+    european_labour_day,
+    assumption_day,
+    all_saints_day,
+    immaculate_conception,
+    christmas_eve,
+    christmas,
+    new_years_eve,
+)
+from .trading_calendar import HolidayCalendar, TradingCalendar
+
+NewYearsDay = new_years_day()
+
+Epiphany = epiphany()
+
+AscensionDay = ascension_day()
+WhitMonday = whit_monday()
+CorpusChristi = corpus_christi()
+
+LabourDay = european_labour_day()
+
+AssumptionDay = assumption_day()
+
+NationalHoliday = Holiday('National Holiday', month=10, day=26)
+
+AllSaintsDay = all_saints_day()
+
+ImmaculateConception = immaculate_conception()
+
+ChristmasEve = christmas_eve()
+Christmas = christmas()
+
+SaintStephensDay = Holiday('Saint Stephens Day', month=12, day=26)
```

Comment:
I think this should be `Stephen's`

Reply:
Also wondering if we could combine this with Boxing Day?

Reply:
> I think this should be Stephen's

Yup I'll fix

> Also wondering if we could combine this with Boxing Day?

I thought of this too but am inclined to say no just because it's a simple enough holiday to just give it its own name. cc @jmccorriston